book/styles.md

```diff
@@ -1071,7 +1071,8 @@ case the existing layout algorithm is used.
 6-3 *Class selectors*. Any HTML element can have a `class` attribute,
 whose value is a space-separated list of that element's classes. A CSS
 class selector, like `.main`, affects all elements with the `main`
-class. Implement class selectors; give them priority 10. If you've
+class. Implement class selectors; they should take precedence over tag
+selectors. If you've
```

Comment:
I thought dictating the priority here was odd. This language dates from an earlier draft where we implemented ID selectors in the main text so we wanted to specify here that they go in between.